cypress/integration/Util.ts

```diff
@@ -0,0 +1,55 @@
+export function createInstance(newInstanceName: string) {
+  cy.ouiaId("create-smart-event-instance", "PF4/Button").click();
+  cy.ouiaId("create-instance", "PF4/ModalContent").then(($modal) => {
+    cy.wrap($modal)
+      .should("be.visible")
+      .within(() => {
+        cy.ouiaId("new-name", "PF4/TextInput").type(newInstanceName);
+        cy.ouiaId("info-instance-available-soon", "PF4/Alert").should(
+          "have.text",
+          "Info alert:Your Smart Events instance will be ready for use shortly after creation."
+        );
+        cy.ouiaId("submit", "PF4/Button").click();
+      });
+    cy.wrap($modal, { timeout: 20000 }).should("not.exist");
+  });
+}
+
+export function newInstanceStatus(instanceName: string) {
```

Comment:
@ChughShilpa I am a little bit confused by name of this function. When I read code I was not sure about its purpose. I think about `waitInstanceIsReady(instanceName)` or `waitInstanceStatus(instanceName, status)`.
It would be nice to have timeout as optional parameter.

@jomarko what is your opinion?

Reply:
@sjamboro yes, your proposal improves the code readabiloity, +1 to follow your proposal

Reply:
Renamed the function name as suggested